tipg/dbmodel.py

```diff
@@ -446,19 +1068,19 @@ async def get_table_index(
             if not geometry_column and geometry_columns:
                 geometry_column = geometry_columns[0]
 
-            catalog[id] = {
-                "type": table["entity"],
-                "id": id,
-                "table": table["tbl"],
-                "schema": table["dbschema"],
-                "description": table["description"],
-                "id_column": id_column,
-                "geometry_columns": geometry_columns,
-                "datetime_columns": datetime_columns,
-                "properties": properties,
-                "datetime_column": datetime_column,
-                "geometry_column": geometry_column,
-                "parameters": table["parameters"],
-            }
+            catalog[id] = Collection(
```

Comment:
We now store `Collection` object directly into the catalog (instead of a dictionary)